src/pharmpy/tools/external/nonmem/results_file.py

```diff
@@ -356,7 +356,9 @@ def log_items(self, lines):
     def tag_items(self, path):
         nmversion = re.compile(r'1NONLINEAR MIXED EFFECTS MODEL PROGRAM \(NONMEM\) VERSION\s+(\S+)')
         tag = re.compile(r'\s*#([A-Z]{4}):\s*(.*)')
-        end_TERE = re.compile(r'(0|1)')  # The part we need after #TERE: will preceed next ^1 or ^0
+        end_TERE = re.compile(
```

Comment:
Why was this line reformatted. Did black do this?

Reply:
Yes, this was reformatted through `tox -e format`

Reply:
Ok, strange that it didn't do that for the previous version.